swagger/v1/framework.yaml

```diff
@@ -34,8 +34,6 @@ Framework:
           readOnly: true
     relationships:
       type: object
-      required:
-      - questions
```

Comment:
I had to make this change to support the `GET /reference/frameworks` endpoint; this renders frameworks without questions which then breaks the JSON validation. The other way around that would be to add a new definition for the alternate lightweight version of framework but this felt cleaner 🤔